sample_files/main.py

```diff
@@ -1,145 +1,80 @@
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+import logging
+import time
+import requests
+from requests.exceptions import RequestException
+from selenium.common.exceptions import WebDriverException
 from langchain_openai import ChatOpenAI
-import os
-import sys
-from langchain_core.prompts import ChatPromptTemplate
-from dotenv import load_dotenv
-from langchain_experimental.utilities import PythonREPL
-from langchain.tools import Tool
 from langchain_core.output_parsers import StrOutputParser
 from utils import sanitize_output, read_urls, html_scrapper
 from sys_prompts import system_prompt
-from logs.logger import logger
-import requests
-from requests.exceptions import RequestException
-import time 
-from selenium import webdriver
-from selenium.webdriver.common.by import By
-from selenium.common.exceptions import WebDriverException
-
-
-# Load environment variables from the .env file where our api key is stored
-load_dotenv()
-
-# fetching the api key from the .env file
-api_k = os.getenv('OPENAI_API_KEY')
-
-# chat model object (default model- gpt3.5 turbo)
-llm = ChatOpenAI(temperature=0.8,api_key=api_k)
-
+from logs.logger import logger  # Assuming this is a custom logger instance
 
-# Prompt for system message
-sys_prompt = system_prompt("Selenium 4.20.0 generator openai")
+# Initialize logging
+logging.basicConfig(level=logging.INFO)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logging configuration is being set up twice, once with `logging.basicConfig(level=logging.INFO)` and then again by importing a custom logger instance. This could lead to confusion and unexpected behavior. It would be better to use only one method of setting up logging.

```diff
- # Initialize logging
- logging.basicConfig(level=logging.INFO)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->